readme.md

```diff
@@ -0,0 +1,237 @@
+```
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --with-local-conf --with-do-build
+```
+
+### Few build hints
+Pay attention that mentioned above command will erase `workspace_base_dir` and start clear build.
+
+* If case of interruption you can just re-run script keeping all fetched sources:
+```
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --continue-build
+```
+
+* You can get "essence" of prod-aos, apply some pathes/modifications and continue build:
+```
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --with-local-conf
+pushd <workspace_base_dir>/meta-xt-prod-devel
+# do something, apply patches
+popd
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --continue-build
+```
+
+* To clear build products and intermediate files:
+```
+rm -rf <workspace_cache_base_dir>
+cd <workspace_base_dir>/build
+# remove everything except conf/ folder
+rm -rf bitbake.lock buildhistory/ cache/ deploy/ log/ shared_rootfs/ tmp/
+```
+
+## How to install
+
+### Write to microSD card
+As already mentioned in Quick start section, you can use `dd` or `bmaptool`.
+```
+sudo dd if=./image.img of="/dev/sd?" bs=1M status=progress
+```
+or bmaptool which is strongly recommended due to higher speed
+```
+sudo bmaptool copy --bmap image.img.bmap ./imgage.img /dev/sd?
+```
+
+### Write to eMMC
+We have two ways to put binaries onto eMMC: using SD card or using TFTP and NFS.
+
+#### Write to eMMC using SD card
+Detailed instruction with manual and scripted option is provided on wiki page
+https://github.com/xen-troops/meta-xt-prod-devel/wiki/How-to-flash-image-to-eMMC-from-SD-card
+
+#### Write to eMMC using network
+We need to assume that you have TFTP and NFS servers working on your PC.
+TODO Add page to wiki and put link here
+
+## Supported boards
+List of supported MACHINE_NAMEs:
+```
+MACHINE_NAME         Board name         SoC       RAM  prod-devel-unsafe only?  DomA supported?
+------------------------------------------------------------------------------------------------------
+salvator-x-h3-4x2g   Salvator-X         H3 ES3.0  8GB
+salvator-x-m3        Salvator-X         M3        4GB  prod-devel-unsafe only   Not supported for DomA
+salvator-xs-h3-4x2g  Salvator-XS        H3 ES3.0  8GB
+salvator-xs-m3n      Salvator-XS        M3N       2GB                           Not supported for DomA
+h3ulcb-4x2g          H3ULCB             H3 ES3.0  8GB
+h3ulcb-4x2g-kf       H3ULCB KingFisher  H3 ES3.0  8GB
+```
+
+## References
+* If you are not aware what is meta layer - you may read Yocto's documentation at https://www.yoctoproject.org/docs/
+* If you want to understand what is hypervisor and guest OS - check https://xenproject.org/help/documentation/
+* To review Renesas boards - check https://www.renesas.com/eu/en/solutions/automotive/adas/solution-kits/r-car-starter-kit.html
+
```

Comment:
I would put something pointing to Yocto BSP located at elinux.org, https://elinux.org/R-Car/Boards/Yocto-Gen3/v5.1.0 or https://elinux.org/Category:R-Car_Gen3, etc

Reply:
Thanks, I have added these links.